test/controllers/NextUpdatesControllerSpec.scala

```diff
@@ -16,76 +16,77 @@
 
 package controllers
 
+import java.time.LocalDate
+
 import assets.BaseTestConstants
-import assets.MessagesLookUp.{NoReportDeadlines, Obligations => obligationsMessages}
+import assets.MessagesLookUp.{NoNextUpdates, Obligations => obligationsMessages}
 import audit.AuditingService
 import config.featureswitch.{FeatureSwitching, NextUpdates}
 import config.{FrontendAppConfig, ItvcErrorHandler}
 import controllers.predicates.{NinoPredicate, SessionTimeoutPredicate}
 import mocks.controllers.predicates.{MockAuthenticationPredicate, MockIncomeSourceDetailsPredicate}
-import mocks.services.MockReportDeadlinesService
-import models.reportDeadlines.{ObligationsModel, ReportDeadlineModel, ReportDeadlinesModel, ReportDeadlinesResponseModel}
+import mocks.services.MockNextUpdatesService
+import models.nextUpdates.{NextUpdateModel, NextUpdatesModel, NextUpdatesResponseModel, ObligationsModel}
 import org.jsoup.Jsoup
 import org.mockito.ArgumentMatchers.{any, eq => matches}
 import org.mockito.Mockito.when
 import org.mockito.stubbing.OngoingStubbing
 import play.api.http.Status
 import play.api.mvc.MessagesControllerComponents
 import play.api.test.Helpers._
-import services.ReportDeadlinesService
-import views.html.{NextUpdates, NoReportDeadlines, Obligations}
+import services.NextUpdatesService
+import views.html.{NextUpdates, NoNextUpdates, Obligations}
 
-import java.time.LocalDate
 import scala.concurrent.Future
 
-class ReportDeadlinesControllerSpec extends MockAuthenticationPredicate with MockIncomeSourceDetailsPredicate
-                                            with MockReportDeadlinesService with FeatureSwitching{
+class NextUpdatesControllerSpec extends MockAuthenticationPredicate with MockIncomeSourceDetailsPredicate
+                                            with MockNextUpdatesService with FeatureSwitching{
 
-  object TestReportDeadlinesController extends ReportDeadlinesController(
-    app.injector.instanceOf[NoReportDeadlines],
+  object TestNextUpdatesController$ extends NextUpdatesController(
```

Comment:
Please rename the object to get rid of the '$' sign at the end

Reply:
Fixed